src/SoftCacheTrait.php

```diff
@@ -17,8 +17,12 @@
 	 * @param $output mixed
 	 */
 	public function writeMethodCache($method, array $args, $output) {
-		$cache_key = $this->getCacheKey($args);
-		$this->cache[$method][$cache_key] = $output;
+		try {
+			$cache_key = $this->getCacheKey($args);
+			$this->cache[$method][$cache_key] = $output;
```

Comment:
I'd like to have one test case. Could you identify which line did throw the exception you found in the logs, and then stub it like here: https://stackoverflow.com/questions/45905183/mocking-methods-to-throw-error-phpunit
You may have to move `$this->cache[$method][$cache_key] = $output;` to a sub function.